### PR TITLE
make rank1d quick method primetime

### PR DESCRIPTION
**Pull request by mchestnut91**
Addresses the rank1d quick method  for #960

I have made the following changes:

Added a `show` parameter to the quick method function signature, as well as to the docstring
Added an image comparison unit test
Added the `rank1d` quick method to the API reference
Edited the summary table in rankd.rst
Added an example usage of the feature_correlation quick method in rankd.rst

### Sample Code and Plot
from yellowbrick.datasets import load_credit
from yellowbrick.features.rankd import rank1d

X, y = load_credit()
fig = rank1d(X, y)

image

### CODE REVIEW

**Comment by rebeccabilbro** [comment]
Hi there @mchestnut91 and thank you very much for the PR!! Just a heads up that I manually cancelled the Appveyor build, so don't worry about the red x! I've been manually cancelling the builds today so that I can orchestrate them to get as many PRs merged this weekend as we can. I'll get it re-started shortly!

**Comment by mchestnut91** [comment]
Hi @rebeccabilbro thanks for letting me know! Best of luck pushing towards release this weekend! 👍

File: tests/test_features/test_rankd.py [hunk 1]
```diff
@@ -295,7 +295,20 @@ def test_rank1d_integrated_numpy(self):
         # Image similarity testing
         oz.finalize()
         self.assert_images_similar(oz)
+    
+    @pytest.mark.xfail(
+        IS_WINDOWS_OR_CONDA,
+        reason="font rendering different in OS and/or Python; see #892",
+    )
```
Comment (rebeccabilbro) [inline tests/test_features/test_rankd.py]: @mchestnut91 @pdamodaran is this xfail necessary? It looks like this test hasn't been through CI yet without the fail, so unless I've missed something, can we remove the xfail decorator and allow the test to run as expected?

Comment (pdamodaran) [reply tests/test_features/test_rankd.py]: @rebeccabilbro - I removed the xfail decorator and successfully reran the tests.

Comment (rebeccabilbro) [reply tests/test_features/test_rankd.py]: thank you!

**Pull request merged**